servisor.js

```diff
@@ -57,17 +58,26 @@ Servisor.prototype.run = function run (conf) {
     });
 };
 
-Servisor.prototype._sanitizeConfig = function (conf) {
+Servisor.prototype._sanitizeConfig = function (conf, options) {
     // TODO: Perform proper validation!
     if (!conf.logging) { conf.logging = {}; }
     if (!conf.metrics) { conf.metrics = {}; }
+    // check the number of workers to run
+    if(options.num_workers !== -1) {
+        // the number of workers has been supplied
+        // on the command line, so honour that
+        conf.num_workers = options.num_workers;
+    } else if(conf.num_workers === 'ncpu') {
```

Comment:
Yeah, I was thinking about that too while coding it and I'm not sure we should. Take for example `options.num_workers = -1` and `conf.num_workers = 0`. Here we would not honour the config directive. What I would add here, though, is a check:
```javascript
  } else if(conf.num_workers === 'ncpu' || typeof conf.num_workers !== 'number') {
```

What do you think about it? This makes us more cautious while still config-abiding if a number is present